lib/elixir/lib/io.ex

```diff
@@ -32,6 +29,109 @@ defmodule IO do
   was last accessed. The position of files can be changed using the
   `:file.position/2` function.
 
+  ## Iolists and iodata
```

Comment:
What do you think about "IO Data/IO Lists" or "IOData/IOLists"? I've seen the latter [in the Erlang/OTP codebase](https://github.com/erlang/otp/blob/master/lib/kernel/src/raw_file_io_list.erl#L74) 🙂

Reply:
Now that I think about it, we don't expose iolists anywhere in Elixir except in the typespecs. Most places where we say iolists informally we actually mean iodata. So my suggestion is to call it "IO data" and focus on that. We can have one note on iolists saying that iolists are the same as iodata except they don't accept a binary in the top level.

Reply:
Good call on dropping the iolists. We should go with "IO data" for the naming IMO.